packages/client/src/components/Modals/SavedSearchPanel.vue

```diff
@@ -34,7 +34,7 @@
                 <b-icon icon="three-dots-vertical" class="text-dark" font-scale="1"></b-icon>
               </template>
               <b-dropdown-item :searchId="search.id" @click.stop="editSavedSearch">Edit</b-dropdown-item>
-              <b-dropdown-item @click="deleteSavedSearch" :searchId="search.id">Delete</b-dropdown-item>
+              <b-dropdown-item @click.stop="deleteSavedSearch" :searchId="search.id">Delete</b-dropdown-item>
```

Comment:
This was why the row was getting applied from the spooky beyond despite being deleted.